gum/backend-darwin/gumprocess-darwin-priv.h

```diff
@@ -0,0 +1,40 @@
+#ifndef __GUM_PROCESS_DARWIN_PRIV_H__
+#define __GUM_PROCESS_DARWIN_PRIV_H__
```

Comment:
This header isn't self-contained, as it uses GLib types without including needed header(s). Would also be good to add the copyright header. (Though I'm thinking about dropping them at some point — but for now let's add it for consistency.)